types/rox-browser/index.d.ts

```diff
@@ -0,0 +1,273 @@
+ */
+export function fetch(): Promise<unknown>;
+
+/**
+ * Opens the flag override view, providing a debug UI for the application's set of feature flags.
+ * https://support.rollout.io/docs/javascript-browser-api#section-showoverrides
+ */
+export function showOverrides(position?: RoxOverridesPosition): void;
+
+export enum RoxOverridesPosition {
+  TopLeft = 'top left',
+  TopRight = 'top right',
+  BottomLeft = 'bottom left',
+  BottomRight = 'bottom right'
+}
+
+/**
+ * Default is untilForeground
+ *
+ * https://support.rollout.io/docs/flags-update-flow#section-flag-freeze-level
+ */
+export enum RoxFlagFreezeLevel {
+  None = 'none',
+  UntilForeground = 'untilForeground',
+  UntilLaunch = 'untilLaunch'
+}
+
+export interface RoxFlagOptions {
+  freeze?: RoxFlagFreezeLevel;
+}
+
+/**
+ * Creates a new Flag
+ * https://support.rollout.io/docs/javascript-browser-api#section--rox-flag-
+ */
+export class Flag {
+  constructor(defaultValue?: boolean, options?: RoxFlagOptions);
+
+  // The name of the Flag
+  readonly name: string;
+
+  // Default value of the Flag
+  readonly defaultValue: boolean;
+
+  // Returns true when the flag is enabled
+  isEnabled(): boolean;
+
+  // Unlock the Flag value from changes from the last time it was freezed
+  unfreeze(): void;
+}
+
+/**
+ * Used to create and manage Rollout feature flags that determine different predefined values
+ *
+ * https://support.rollout.io/docs/javascript-browser-api#section--rox-variant-
+ */
+export class Variant<T extends string = string> {
+  constructor(defaultValue: T, options: ReadonlyArray<T>, name?: string);
+
+  // The name of the Variant
+  readonly name: string;
+
+  // Default value of the Variant
+  readonly defaultValue: BasicType<T>;
+
+  // Returns the current value of the Variant, accounting for value overrides
+  getValue(): BasicType<T>;
+
+  // Unlock the Variant value from changes from the last time it was freezed
+  unfreeze(): void;
+}
+
+/**
+ * manages a remote configuration setting with a value of type string, boolean, or number.
+ * The constructor sets the default value for the remote configuration setting
+ *
+ * https://support.rollout.io/docs/javascript-browser-api#section--rox-configuration-
+ */
+export class Configuration<T extends number | boolean | string> {
+  constructor(defaultValue: T);
+
+  // The name of the Configuration
+  readonly name: string;
+
+  // Default value of the Configuration
+  readonly defaultValue: BasicType<T>;
+
+  // Returns the current value of the Configuration, accounting for value overrides
+  getValue(): BasicType<T>;
+
+  // Unlock the Configuration value from changes from the last time it was freezed
+  unfreeze(): void;
+}
+
+/**
+ * Ensure that TypeScript properly types things with a basic type.
+ * For example, if T is true, returnedtype shall be boolean, not true
+ */
+type BasicType<T> = T extends boolean ? boolean : T extends number ? number : T extends string ? string : never;
```

Comment:
Unless you explicitly do not want this to be exported, you can remove the `export {}` on line 14.

Reply:
I've remove `export {}`